src/store/state.js

```diff
@@ -0,0 +1,67 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const state = createSlice({
+  name: 'state',
+  initialState: {
+    activityIndex: 0,
+    adIndex: 0,
+    boardMessageIndex: 0,
+    availableQuotes: [],
+    usedQuotes: [],
+    quoteIndex: 0,
+    current: 'activities'
+  },
+  reducers: {
+    incrementActivityIndex(state) {
+      state.activityIndex++;
+    },
+    resetActivityIndex(state) {
+      state.activityIndex = 0;
+    },
+
+    incrementAdIndex(state) {
+      state.adIndex++;
+    },
+    resetAdIndex(state) {
+      state.adIndex = 0;
+    },
+
+    incrementBoardMessageIndex(state) {
+      state.boardMessageIndex++;
+    },
+    resetBoardMessageIndex(state) {
+      state.boardMessageIndex = 0;
+    },
+
+    nextQuote(state) {
```

Comment:
should this not be in a seperate Reducer? I would have created a seperate reducer for quotes.

In the store/state.js file you would like to keep the logic side as slim (thin / small) as possible and not have any logic of entities as quotes.

Reply:
I've thought about it for a bit, and i think i agree: i'm going to separate the quotes logic into its own reducer and the leave the remaining logic as the "screen reducer".